browser-extension/common/src/custom-elements/test-rule-widget/index.ts

```diff
@@ -62,13 +80,14 @@ export abstract class RQTestRuleWidget extends RQDraggableWidget {
             <button id="test-rule-minimized-btn"></button>
         </div>
         <div id="heading-container">
-            <div id="logo-container"> 
+            <div id="logo-container">
                 <span id="heading-logo">${RQLogo}</span>
                 <span id="logo-text">requestly</span>
             </div>
             <div id="actions-container">
-                <button id="settings-button" class="hidden">${SettingsIcon}</button>
+                <button id="settings-button" class="hidden" data-tooltip="Hide widget in app settings">${VisibilityIcon}</button>
                 <button id="minimize-button">${MinimizeIcon}</buttton>
+                <button id="close-button" class="hidden">${CloseIcon}</buttton>
```

Comment:
This is common to both the explicit and implicit widgets. Can we keep the components being used in only one of them specfic to its class? It will be cleaner instead of managing its visibility using javascript

Reply:
Since its a close button which we might be used in both type of widgets, so that’s why keeping it here only.